src/task 20.1.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Katalog `src` to nie jest najlepsze miejsce na przechowywanie plików HTML. 
Dopóki to nie jest wpięte w żaden sposób z kontrolerami, proponuję `src/main/resources/html`.
Ogólnie katalog `resources` jest przeznaczony dla plików, które nie są kodem.